index.js

```diff
@@ -129,18 +128,14 @@ Client.registry.registerCommands([
   require('./commands/raids/auto-status'),
   require('./commands/raids/report-privacy'),
 
-  require('./commands/trains/train'),
```

Comment:
Just a note for us, there's a bit of train functionality that was removed and something that I'll handle manually merging you're changes in. :) Still gotta go over the new stuff you added.